wrappers/android/examples/java_example/build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.2.1'
+        classpath 'com.android.tools.build:gradle:4.2.1'
```

Comment:
with 4.2.1, jcenter() is deprecated, replace it with mavenCentral() in librealsense\wrappers\android\examples\java_example\build.gradle to remove the compilation warning.

Reply:
Done